src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -205,10 +200,70 @@ class ProjectReview extends Component {
 
   }
 
-  approveEdits = (e) => () => {
+  approveEdits() {
+    let project = this.getProject();
+    Project.updateProject(this.props.updateProjectUrl, project, this.props.projectKey).then(
+      resp => {
+        window.location.href = [this.props.serverURL, "index"].join("/");
+        spinnerService.showAll();
+      })
+      .catch(error => {
+      spinnerService.hideAll();
+      console.error(error);
```

Comment:
nit: indentation, here and line above
```suggestion
        console.error(error);
```